lib/game_completion.rb

```diff
@@ -6,14 +6,12 @@ def initialize(console_ui, game_eval)
       @game_eval = game_eval
     end
 
-    def game_over_messages(board_array, marker1, marker2)
+    def game_over_messages(board_array, player1, player2)
       @console_ui.display_board(board_array)
       @console_ui.puts_space
-      game_result = @game_eval.player_won_or_tied(board_array, marker1, marker2)
-      if game_result ==  marker1 
-        @console_ui.won_message("Player 1")
-      elsif game_result == marker2
-        @console_ui.won_message("Player 2")
+      player_won = @game_eval.player_won_or_tied(board_array, player1, player2)
```

Comment:
`player_won` makes me think this variable is a boolean. What about naming it something like `winning_player`?